docs/sources/administration/configuration.md

```diff
@@ -1101,7 +1101,7 @@ Sets a global limit on number of alert rules that can be created. Default is -1
 
 ## [unified_alerting]
 
-For more information about the Unified Alerting feature in Grafana, refer to [Unified Alerting]({{< relref "../unified-alerting/_index.md" >}}}).
+For more information about the Grafana 8 alerts, refer to [Unified Alerting]({{< relref "../alerting/unified-alerting/_index.md" >}}).
```

Comment:
From a maintenance point of view - do we really want to brand the new alerting structure as Grafana 8 alerts? Seems like it'll just have to change for each release...this probably isn't the place to discuss this, but I thought I'd point it out, less I forget.